unity/unity.go

```diff
@@ -0,0 +1,115 @@
+package unity
+
+import (
+	"bytes"
+	"embed"
+	"html/template"
+	"net/http"
+	"utils"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	CHARSET = "utf-8"
+)
+
+type UnityLeaveData struct {
+	Name     string `json:"name" form:"name"`
+	Date     string `json:"date" form:"date"`
```

Comment:
Need to take 2 dates, `start_date` and `end_date`

Reply:
have you discuss with jimmy sir that should we take two dates?